ephios/templates/base.html

```diff
@@ -72,9 +72,11 @@
         <header>
             <nav class="navbar navbar-expand-lg navbar-light bg-light shadow">
                 <div class="container">
-                    <a class="navbar-brand" href="/"><img src="{% static "ephios/img/ephios-text-black.png" %}"
-                                                          alt="ephios"
-                                                          class="d-inline-block" width="75"></a>
+                    <a class="navbar-brand" href="/">
```

Comment:
signal to customize the href would be nice for federation (can be a separate issue)

Reply:
You could specify the href using the attrs attribute 🤔